opsmngr/automation_config.go

```diff
@@ -236,6 +236,11 @@ type ReplicaSet struct {
 	Members                            []Member                `json:"members"`
 	Settings                           *map[string]interface{} `json:"settings,omitempty"`
 	WriteConcernMajorityJournalDefault string                  `json:"writeConcernMajorityJournalDefault,omitempty"`
+	Force                              Force                   `json:"force,omitempty"`
```

Comment:
[q] do we need to make this a pointer so it can be null?
```suggestion
	Force                              *Force                   `json:"force,omitempty"`
```